### PR TITLE
Replace setTimeout with step_timeout

### PR DESCRIPTION
**Pull request by plehegar**
Minor change on a manual test, to promote best practices

### CODE REVIEW

File: pointerevents/pointerevent_boundary_events_at_implicit_release_hoverable_pointers-manual.html [hunk 1]
```diff
@@ -42,12 +42,10 @@
             detected_pointertypes[event.pointerType] = true;
             event_log.push(event.type + '@' + event.target.id);
             if (event.type == 'lostpointercapture') {
-              setTimeout(function() {
-                test_pointer_event.step(function () {
+              test_pointer_event.step_timeout(function () {
                   var expected_events = "pointerup, lostpointercapture, pointerout, pointerleave";
```
Comment (Steditor) [inline pointerevents/pointerevent_boundary_events_at_implicit_release_hoverable_pointers-manual.html]: I think we should then adjust the indentation in 46 and 47.

Comment (plehegar) [reply pointerevents/pointerevent_boundary_events_at_implicit_release_hoverable_pointers-manual.html]: good catch. fixed.

**Review by Steditor — CHANGES_REQUESTED** [review]
Currently this results in `TypeError: test_pointer_event.step_timeout is not a function` for me - and as far as I can see, `setup_pointerevent_test` returns a `MultiPointerTypeTest` from `pointerevent_support.js`, which is never equipped with a `step_timeout`-function.

**Comment by plehegar** [comment]
you're right. this is more work than worth it to remove that one setTimeout, and for a manual test. abandoning this.

**Pull request closed without merging**